.travis.rosinstall

```diff
@@ -1,12 +1,20 @@
 - git:
     uri: 'https://github.com/ipa320/cob_common.git'
     local-name: cob_common
-    version: indigo_dev
+    version: kinetic_dev
 - git:
     uri: 'https://github.com/ipa320/cob_driver.git'
     local-name: cob_driver
-    version: indigo_dev
+    version: kinetic_dev
 - git:
     uri: 'https://github.com/ipa320/cob_extern.git'
     local-name: cob_extern
-    version: indigo_dev
+    version: kinetic_dev
+- git:
+    uri: 'https://github.com/ros-industrial/ros_canopen.git'
+    local-name: ros_canopen
+    version: jade-devel
```

Comment:
yes, this is the latest version from ros_canopen. It should compile under kinetic